static/routes.config.php

```diff
@@ -387,6 +387,7 @@
 		'/hidden'                     => [Module\Contact::class,           [R::GET]],
 		'/ignored'                    => [Module\Contact::class,           [R::GET]],
 		'/hovercard'                  => [Module\Contact\Hovercard::class, [R::GET]],
+		'/follow[/{url}]'             => [Module\Contact\Follow::class,    [R::GET, R::POST]],
```

Comment:
Parameter is never used in the module.
```suggestion
		'/follow'                     => [Module\Contact\Follow::class,    [R::GET, R::POST]],
```

Reply:
it is, why not?

Reply:
ah, it's a GET parameter, I'm an idiot, thx